src/badguy/ghoul.hpp

```diff
@@ -43,6 +43,8 @@ class Ghoul final : public BadGuy,
   void stop_moving();
 
   void move_to(const Vector& pos) override;
+
+  Path* get_path_if_exists() override { return get_path(); }
   

```

Comment:
`get_path()` is already public; you may use it directly instead.

Generally, it's not a good idea to add "if_exists" to the names of functions that return a pointer; if a pointer is returned, it is already assumed that the pointer might be null.